### ISSUE TITLE
Frontend not runnable (Node LTS 16->18, OpenSSL error)

### ISSUE THREAD

**Issue by BorysekOndrej** [pwndoc/pwndoc#383]
Hi, due to update in Node version (16 ->18) any new build of Frontend errors-out on start.
This problem is not specific to just to PwnDoc ([relevant StackOverflow post](https://stackoverflow.com/q/69665222)), however it seems that this might not get resolved upstream.

Workaround is to modify `frontend/Dockerfile` (and potentially `frontend/Dockerfile.dev`):
`FROM node:lts-alpine AS build` -> `FROM node:16-alpine AS build`

Node 18 [became LTS](https://github.com/nodejs/Release#release-schedule) on 2022-10-25.

Excerpts from trace:

```js
[  Error: error:0308010C:digital envelope routines::unsupported

{
  opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
  library: 'digital envelope routines',
  reason: 'unsupported',
  code: 'ERR_OSSL_EVP_UNSUPPORTED'
}
```

<details>
  <summary>Full trace</summary>

```js

Node.js v18.12.0
error Command failed with exit code 1.
/app/node_modules/loader-runner/lib/LoaderRunner.js:133
                if(isError) throw e;
                            ^

[  Error: error:0308010C:digital envelope routines::unsupported
  
  - hash:71 new Hash
    node:internal/crypto/hash:71:19
  
  - node:crypto:133 Object.createHash
    node:crypto:133:10
  
  - createHash.js:135 module.exports
    [app]/[webpack]/lib/util/createHash.js:135:53
  
  - NormalModule.js:417 NormalModule._initBuildHash
    [app]/[webpack]/lib/NormalModule.js:417:16
  
  - NormalModule.js:471 handleParseError
    [app]/[webpack]/lib/NormalModule.js:471:10
  
  - NormalModule.js:503 
    [app]/[webpack]/lib/NormalModule.js:503:5
  
  - NormalModule.js:358 
    [app]/[webpack]/lib/NormalModule.js:358:12
  
  - LoaderRunner.js:373 
    [app]/[loader-runner]/lib/LoaderRunner.js:373:3
  
  - LoaderRunner.js:214 iterateNormalLoaders
    [app]/[loader-runner]/lib/LoaderRunner.js:214:10
  
  - LoaderRunner.js:221 iterateNormalLoaders
    [app]/[loader-runner]/lib/LoaderRunner.js:221:10
  
] {
  opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
  library: 'digital envelope routines',
  reason: 'unsupported',
  code: 'ERR_OSSL_EVP_UNSUPPORTED'
}
```
</details>

**The PR to fix this issue is #384. It (and other PRs) are also merged in the fork [pwndoc-ng](https://github.com/pwndoc-ng/pwndoc-ng).**

**Comment by jasj23** [comment]
I am having the same problem... any solution please?

**Comment by MaxNad** [comment]
Fixed in the [9d0c518](https://github.com/pwndoc/pwndoc/commit/9d0c5184bc662a9be32a18e285e3308d5f473b64) commit
